README.md

```diff
@@ -1,4 +1,5 @@
 [![CircleCI](https://dl.circleci.com/status-badge/img/gh/sambanova/generative_data_prep/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/sambanova/generative_data_prep/tree/main)
+[![codecov](https://codecov.io/gh/sambanova/generative_data_prep/graph/badge.svg?token=9CYRCUOOAO)](https://codecov.io/gh/sambanova/generative_data_prep)
```

Comment:
<img width="754" alt="image" src="https://github.com/user-attachments/assets/ea6b3abb-8433-4f1a-941d-ce9fb484296d">
Looks like this banner is not displaying properly?

Reply:
That'll update once this merges to main since it shows the status of `main`